examples/enums/enums.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/io;
+
+// An enum is defined using a module enum declaration.
+enum Color {
+  RED,
+  GREEN,
+  BLUE
+}
+
+// An enum member can explicitly specify a value in the enum declaration.
+enum Language {
+    ENG = "English",
```

Comment:
On second thoughts, shall we move this comment to L12?

```suggestion
enum Language {
    // An enum member can explicitly specify a value in the enum declaration.
    ENG = "English",
```

Reply:
I think it better if it was above, as this comment also applies for the next enum declaration.

Reply:
But that's OK right we explain the concept here and then use it in a subsequent snippet?

Maybe we can add another comment for `Bands` like
```
// An enum declaration can consist of a combination of members with or without 
// explicit associated values.
```

@pubudu91 any thoughts?

Reply:
+1, changed.